### DISCUSSION TITLE
Extra empty pages after list of tables

### DISCUSSION THREAD

**Discussion by fb-correia** [joaomlourenco/novathesis#318] [Q&A] [answered]
Hello, 

after the list of tables I have 2 extra empty pages. How can I solve this problem?
Thank you

**Comment by joaomlourenco** [comment]
@fb-correia if you do not present/describe properly your context, it is hard to help you.

1) Which version of the template are you using?
2) Local or online (overleaf)?
3) What do you have before the 2 empty pages? List of tables, I believe…
4) What do you have after the 2 empty pages?
5) Do you have a Glossary? Or a list of Symbols? Or list of Acronyms?  If not, are you "adding" some of these files in “3_files.tex”?

**Reply by joaomlourenco** [reply]
BTW, if the problem is what I believe (by pure speculation/guessing of your problem), it shall be solved in the latest version of the template.

**Reply by fb-correia** [reply]
I'm am using the 6.9.5 version of the template and I'm working online. 
Before the two empty pages I have a list of tables and after that I have a list of acronyms. 

In the "3_files.tex" I have this:
![acrónimos](https://user-images.githubusercontent.com/98908443/157649847-2ce4f055-b5d8-4533-8651-d471b67fdd03.jpg)

Thank you

**Comment by joaomlourenco** [comment, marked as answer]
Please update to the last version of the template.  It shall be fixed.  
Please let us know it it solved (or not) the problem for you.
Thanks.

**Reply by fb-correia** [reply]
Problem solved, thank you
